### DISCUSSION TITLE
Sidebar menu takes up space constantly vs hamburger menu

### DISCUSSION THREAD

**Discussion by DoubleStrike** [Rem0o/FanControl.Releases#1864] [General]
For those of us that like to keep windows small and tidy, the new menu just eats up more space and cannot be collapsed.

Is there a way to allow it to collapse like the old hamburger menu?  I get that it's not the more modern UI style, but frankly, the more modern UI style of wasted space just sucks.

At least an option/icon/menu option to collapse or hide that area would be much appreciated.

**Comment by Rem0o** [comment]
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/409dedea-08d3-48e1-84a2-9d83d6f9fb6b)![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/4ad81bf9-5b9f-4034-acc0-5f420f457fc1)

I'm experimenting combining the old with the new.  Would default to opened for new users, but then it will remember the on/off position.  I think this is a better overall solution and still a clear improvement from the previous side modal menu.
